### PR TITLE
fix(dropdown): label placement and padding

### PR DESCRIPTION
**Pull request by mJarsater**
**Describe pull-request**  
Changed the bottom padding for the button if the label is inside, also refactored the positioning of the actual inside label. Fixes for both components and tegel.

***NOTE***
I don't know if this is something we want to address when we refactor this components instead of these fixes?

**Solving issue**  
Fixes: [AB#2463](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2463) #550

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Dropdown
3. Check the placement of the inside label and bottom padding of the button when there is an inside label.

**Screenshots**  
-

**Additional context**  
-

### CODE REVIEW

**Comment by EmelieLitwin** [comment]
The inside label specs are hard to review but it looks good! 👍 
I think the label and the chevron are grey-958 instead of grey-700, maybe that could be fixed in this pr?

File: tegel/src/components/dropdown/dropdown.scss [hunk 1]
```diff
@@ -56,6 +56,15 @@
     display: flex;
     align-items: center;
 
+    &.sdds-dropdown-toggle-label-inside {
+      &-lg {
+        padding-bottom: 16px;
+      }
+
+      &-md {
+        padding-bottom: 12px;
```
Comment (theJohnnyMe) [inline tegel/src/components/dropdown/dropdown.scss:65]:
```suggestion
        padding-bottom: 16px;
```

File: components/src/components/dropdown/dropdown.scss [hunk 2]
```diff
@@ -49,6 +49,16 @@
...
     display: flex;
     align-items: center;
+
+    &.sdds-dropdown-toggle-label-inside {
+      &-lg {
+        padding-bottom: 16px;
+      }
+
+      &-md {
+        padding-bottom: 12px;
```
Comment (theJohnnyMe) [inline components/src/components/dropdown/dropdown.scss:59]:
```suggestion
        padding-bottom: 16px;
```

**Review by theJohnnyMe — CHANGES_REQUESTED** [review]
When there is no label inside and the component size is "Medium", the text is not in the vertical center. I tested by changing the values of padding to 16px and it looks like one in the Figma file. Please check if that would work for you.

**Comment by mJarsater** [comment]
> When there is no label inside and the component size is "Medium", the text is not in the vertical center. I tested by changing the values of padding to 16px and it looks like one in the Figma file. Please check if that would work for you.

Nice catch! I updated the PR and moved all the padding styling from our dropdown-core.scss to dropdown.scss in order to handle sizes better. Please give it a look 😄

**Comment by mJarsater** [comment]
> @mJarsater - Please double-check the native story, it seems that one needs an update with sizing classes now.

Should be fixed now. Still pondering if we should release this as a fix or just skip it and fix it once we refactor the component?

**Review by theJohnnyMe — APPROVED** [review]
LGTM

**Comment by theJohnnyMe** [comment]
> > @mJarsater - Please double-check the native story, it seems that one needs an update with sizing classes now.
> 
> Should be fixed now. Still pondering if we should release this as a fix or just skip it and fix it once we refactor the component?

If there is anything left, please add it as a ticket so we do not forget about it.

**Pull request merged**